extension/src/experiments/columns/like.ts

```diff
@@ -0,0 +1,22 @@
+import { Column } from '../webview/contract'
+
+export type ColumnLike = { label: string; path: string; types?: string[] }
```

Comment:
[F] This definition is used in the quick pick. It means that we don't have to create a full mocked `Column` for "starred".